### PR TITLE
fix: show resource statistics about current resource group

### PR DESCRIPTION
**Pull request by dan-2ee**
## Fix
I just added a resource group selection to the summary page. (The design is the same as the Resource Group checkbox on the Session page.)
Now, you can view resources by selecting a resource group on the Summary page.

## Change
<table>
<tr>
<td><strong>Before</strong></td>
<td><strong>After</strong></td>
</tr>
<tr>
<td>before</td>
<td>after</td>
</tr>
</table>
issue: #1124

### CODE REVIEW

File: src/components/backend-ai-resource-monitor.ts
Comment (inureyes) [inline src/components/backend-ai-resource-monitor.ts]: It should be kept to original implementation. If you want to show the panel on another view, it is better to change the CSS part, not the class casting.

File: src/components/backend-ai-session-view.ts
Comment (inureyes) [inline src/components/backend-ai-session-view.ts]: Please explain why you cast the 'direction' on resource monitor.

**Review by inureyes — CHANGES_REQUESTED** [review]

File: src/components/backend-ai-resource-monitor.ts
Comment (dan-2ee) [reply src/components/backend-ai-resource-monitor.ts]: I accepted the feedback and modified it to leave the original implementation as it is. Please review it again 😃
@inureyes

File: src/components/backend-ai-session-view.ts
Comment (dan-2ee) [reply src/components/backend-ai-session-view.ts]: The initial code was written to show "scaling-group-select-box" in the resource monitor only if the direction is vertical.
So I also set direction to vertical to show the select box on the session page.
However, since the select box is shown everywhere using the resource monitor, I modified it to show the select box by default, not by direction.

Comment (dan-2ee) [reply src/components/backend-ai-session-view.ts]: Also, unlike other pages (summary, import page), the session page displays the select box horizontally. So I changed the following items.
1. Added css code when direction is horizontal.
2. Change the updateScalingGroupSelector() to operate regardless of direction so that the scaling group can be updated even when the direction is horizontal (Previously, it only works when direction is vertical)

**Review by lizable — CHANGES_REQUESTED** [review]
Let's remove spaces around resource monitor panel in session page to keep consistency of the layout.
Perhaps like this?
Screen Shot 2022-07-11 at 8 32 28 PM

File: src/components/backend-ai-resource-monitor.ts
Comment (agatha197) [inline src/components/backend-ai-resource-monitor.ts]: Please remove this blank line.

File: src/components/backend-ai-resource-monitor.ts
Comment (agatha197) [inline src/components/backend-ai-resource-monitor.ts]: Same here.

File: src/components/backend-ai-resource-monitor.ts
Comment (agatha197) [inline src/components/backend-ai-resource-monitor.ts]: Same here.

File: src/components/backend-ai-resource-monitor.ts
Comment (agatha197) [inline src/components/backend-ai-resource-monitor.ts]: Same here.

File: src/components/backend-ai-session-view.ts
Comment (agatha197) [inline src/components/backend-ai-session-view.ts]: I think that it's not necessary to cast to horizontal. Because you modified it to show the scaling group select box regardless of direction.

**Review by Sujin-Kim1 — CHANGES_REQUESTED** [review]

File: src/components/backend-ai-resource-monitor.ts
Comment (dan-2ee) [reply src/components/backend-ai-resource-monitor.ts]: I applied it

File: src/components/backend-ai-resource-monitor.ts
Comment (dan-2ee) [reply src/components/backend-ai-resource-monitor.ts]: I applied it

File: src/components/backend-ai-resource-monitor.ts
Comment (dan-2ee) [reply src/components/backend-ai-resource-monitor.ts]: I applied it

File: src/components/backend-ai-resource-monitor.ts
Comment (dan-2ee) [reply src/components/backend-ai-resource-monitor.ts]: I applied it

File: src/components/backend-ai-session-view.ts
Comment (dan-2ee) [reply src/components/backend-ai-session-view.ts]: thank you. I removed the direction cast part. 😁

File: src/components/backend-ai-resource-monitor.ts
Comment (agatha197) [inline src/components/backend-ai-resource-monitor.ts]: Please remove the blanks.

**Review by Sujin-Kim1 — CHANGES_REQUESTED** [review]

**Review by lizable — CHANGES_REQUESTED** [review]
There are two things to be fixed before merging this PR to main branch.

1. Please update for each component to be vertically aligned. For now `lablup-slider` element in resource statistics panel is slightly moved to the right compared to other elements.
Here are screenshots that refer the issue that I mentioned above:
Screen Shot 2022-07-14 at 4 04 43 PM

Screen Shot 2022-07-14 at 4 05 12 PM

2. Please check whether "see resource monitor" works properly in this branch.
I found that there's a regression in "see resource monitor" switch. Unlikely in main branch, It works OK.
### "see resource monitor" switch in main branch
see-resource-monitor mov

**Review by lizable — CHANGES_REQUESTED** [review]
Let's auto-toggle when the width changes. For now, when a resource gauge showing toggle element appears, the initial status is "off", whereas resource gauges are visible.
Screen Recording 2022-07-15 at 2 04 37 PM mov

Please refer to the code below and apply it.
https://github.com/lablup/backend.ai-webui/blob/1ddb167b53d3e17c42f26d8f8711f9af0661cacf/src/components/backend-ai-webui.ts#L259-L263

**Review by lizable — APPROVED** [review]
I think we need slight cosmetic changes. I'll merge this PR after applying them.
- [x] Removing extraneous color style applied in resource gauge toggle button
- [x] Hide/Show not only for the element but also the space of resource gauges
- [x] Forced showing resource gauges when resource statistics panel is in `horizontal` mode.

## Screenshots
resource-gauge-toggle-button-visibility mov

**Pull request merged**
